crates/uv/tests/pip_install.rs

```diff
@@ -175,7 +175,7 @@ dependencies = ["flask==1.0.x"]
     ----- stdout -----
 
     ----- stderr -----
-    error: Failed to download and build: `project @ file://[TEMP_DIR]/path_dep`
+    error: Failed to build: `project @ file://[TEMP_DIR]/path_dep`
       Caused by: Failed to build: `project @ file://[TEMP_DIR]/path_dep`
```

Comment:
Unfortunately, we have a duplicate "Failed to build" here because of `uv-distribution::Error::Build` — I'm exploring a way to avoid that.

Reply:
https://github.com/astral-sh/uv/pull/6075/commits/6cf1a949806c079dde6fdd94374eb7f503810ffb

It becomes a little ambiguous where we failed for some errors, but I think it's a net improvement.